pygfx/objects/_base.py

```diff
@@ -173,6 +166,15 @@ def __init__(
         self.cast_shadow = False
         self.receive_shadow = False
 
+    @callback
+    def _update_uniform_buffers(self, transform: AffineTransform):
+        self.uniform_buffer.data["world_transform"] = self.world.matrix.T
+        self.uniform_buffer.data["world_transform_inv"] = self.world.inverse_matrix.T
+        self.uniform_buffer.update_range()
+
+        for child in self.children:
+            child._update_uniform_buffers(transform)
```

Comment:
if this callback is also triggered on child elements, won't you be performing redundant work?

Reply:
True. We can actually remove this. It's an artifact of an earlier version of the mechanism I used to propagate updates of the transform from parents to children and I forgot to delete it.